.github/workflows/ci.yml

```
@@ -0,0 +1,77 @@
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.25.1'
+
+      - name: Run golangci-lint
+        uses: golangci/golangci-lint-action@v4
+        with:
+          version: v1.60.2
+          args: --timeout=5m
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)